### PR TITLE
MACACO UPDATE? MACACO UPDATE.

### PR DESCRIPTION
**Pull request by Raze500**
UMA DELICIA.

## Description
ADDS MACACO SOUP TO THE HANDCRAFTEABLE RECIPES!
ADDS MACACO SPRITES TO SAY ''UMA DELICIA'' TO YOUR **RP NEEDS.**

## Motivation and Context
I'M 30% MURICA GUY, 30% JAJA'S HOMBRE AND 40% HUEHUE U BR?

## How Has This Been Tested?
TASTES GOOD!
## Screenshots (if appropriate):

image

Spriter: @Kabra

### CODE REVIEW

**Comment by Nicaragua1** [comment]
Im sorry but i find this a great idea, poorly made. This sprites wastes a lot of potential for such food

A sopa du macaco uma delicia needs arms, and monkey stuff floating on the soup. Not a single head that almost look alive just copy paste from the monkey sprite

**Comment by Raze500** [comment]
Absolute macacous idea, though im not the spriter, if he sees this, is up to him to update the sprite, thanks macaco friend!

**Comment by improvedname** [comment]
image

**Comment by Raze500** [comment]
It just give what is tastes, i dunno what else you wish to change it.

**Comment by improvedname** [comment]
VEGETABLES? Theres more monkey then soup

File: code/modules/food_and_drinks/food/snacks_soup.dm [hunk 1]
```diff
@@ -225,3 +225,14 @@
...
+
+//SOPA DO MACACO SOPA DO MACACO FALLOUT 13
+
+/obj/item/reagent_containers/food/snacks/soup/macaco
+	name = "Macaco soup"
+	desc = "To think, the monkey would've beat you to death and steal your gun."
+	icon_state = "macaco"
+	bonus_reagents = list("nutriment" = 1, "omnizine" = 5, "vitamin" = 5)
+	tastes = list("Monkey" = 1)
+	foodtype = VEGETABLES
```
Comment (UndeadNappist) [inline code/modules/food_and_drinks/food/snacks_soup.dm]: MEAT | VEGETABLES

**Pull request closed without merging**